packages/front-end/components/Experiment/TabbedPage/TrafficAndTargeting.tsx

```diff
@@ -92,6 +92,11 @@ export default function TrafficAndTargeting({
                     </HashVersionTooltip>
                   }
                 </div>
+                {experiment.disableStickyBucketing ? (
+                  <div className="mt-1">
+                    Sticky bucketing: <em>disabled</em>
+                  </div>
+                ) : null}
```

Comment:
This looks fine, but not sure if you meant to have it in this PR

Reply:
oops, thanks will fix